doc/admin/installation.md

```diff
@@ -68,7 +90,7 @@ maintained in the Docker files:
 
 The command lines to install the required packages can be extracted from
 the Docker files by removing the "RUN" command, which is specific to Docker.
-Check the [utils/docker](https://github.com/daos-stack/daos/tree/master/utils/docker)
```

Comment:
The change of [docker] to [utils/docker] from #3328 got reverted here .. why?  Shouldn't we document the full path to the dir instead of just the last piece?

Reply:
This has been updated.